client/src/modules/Mentor/components/TaskStatusTabs/TaskStatusTabs.tsx

```diff
@@ -1,9 +1,9 @@
 import { Tabs } from 'antd';
 import { FC, useMemo } from 'react';
 import { tabsRenderer } from './renderers';
-import { StudentTaskSolutionItemStatus } from '../../constants';
+import { SolutionItemStatus } from '../../constants';
 
-export type Status = StudentTaskSolutionItemStatus;
+export type Status = SolutionItemStatus;
```

Comment:
I don't get a purpose of this re-export and rename, please consider to use it as `SolutionItemStatus`

Reply:
Previously enum name was too long and this renaming was added for reducing the word, but I refactored it later and forgot remove this one. Thank you